cmd/updateenvironment.go

```diff
@@ -38,7 +38,7 @@ var updateenvironmentCmd = &cobra.Command{
 	Short: "Update a new environment",
 	Long: AddAppName(`Update a new environment
     Example:
-    $AppName update environment env1 --organization org --file ./somepath.json`),
+    $AppName update environment env1 --organization org --file ./somepath.json --artifacts ./artifactspath.json`),
```

Comment:
also we have ./somepath.json but it should be yaml, currently only yaml is supported.